src/dependencies.jl

```diff
@@ -722,14 +722,16 @@ function check_system_handle!(ret,dep,handle)
 end
 
 # Default installation method
-if Compat.Sys.isapple()
-    defaults = [Binaries,PackageManager,SystemPaths,BuildProcess]
-elseif Compat.Sys.islinux() || (Compat.Sys.isbsd() && !Compat.Sys.isapple())
-    defaults = [PackageManager,SystemPaths,Binaries,BuildProcess]
+defaults = if Compat.Sys.isbsd()
+    [Binaries, PackageManager, SystemPaths, BuildProcess]
```

Comment:
I don't think Binaries are safe to enable for non-Apple bsd here.

Reply:
Fixed in https://github.com/JuliaLang/BinDeps.jl/pull/334